enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,466 @@
+# New Placement APIs
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The proposed work would define new placement APIs to select a set of managed clusters from multiple managed clustersets.
+
+## Motivation
+
+Since placementrule is used in multiple components (GRC/App/Observability). It is more appropriate to  define it in API repo and have a generic component to consume it based on the current placementrule APIs in https://github.com/open-cluster-management/multicloud-operators-placementrule 
+
+The placement APIs will provide a definition on how to select a set of managed clusters in multiple managed clustersets. And placement decisions will be generated according to the spec defined in the placement APIs.
+
+Placement APIs can be used by a higher primitive and controller to decide which clusters the manifests should be placed.  
+
+
+### Goals
+
+- Define a new placement APIs in API repo to select a set of managed clusters in multiple managed clustersets;
+- Have a generic component to consume placements and generate placement decisions;
+- Limit the access to managed clusters by leveraging managed clustersets;
+- Replace and deprecate the current placementrule APIs in https://github.com/open-cluster-management/multicloud-operators-placementrule
+
+### Non-Goals
+
+- Support tolerations/taints. ManagedCluster has no taint defined in spec yet;
+- Create metrics data when creating/updating placement decisions. It needs further investigation and should be covered in a separate enhancement; 
+
+## Proposal
+
+### User Stories
+
+#### Story 1: A cluster-admin on hub can place a policy or workload to all clusters in all clustersets; The cluster-admin on hub can also place a policy or workload to any selected clusters in clustersets.
```

Comment:
I think we should use  ManagedClusterSets to replace clusterset all over the doc.

Reply:
Done.